### DISCUSSION TITLE
Help for RNA velocity analysis

### DISCUSSION THREAD

**Discussion by yl315504** [satijalab/seurat#7199] [Q&A] [answered]
Hi All,
I am new to this topic. We need to do RNA velocity analysis based on the results from Seurat. 
I know there are some tools, such as velocyto. But it seems I need to generate counts tables (loom) for spliced and unspliced transcripts .
Can I generate loom files from a Seurat object?

Thanks for your suggestions.

**Comment by f6v** [comment, marked as answer]
If I'm correct, you're confusing two different analysis levels. Velocity is calculated from BAM files, i.e. alignment of reads to reference. Seurat works with feature-barcode count matrices, i.e. several steps forward. So, Seurat isn't something you can use to generate spliced/unspliced counts. To get those, you need BAM files from CellRanger output (in case of 10x data).

**Reply by yl315504** [reply]
Thank you so much for your response. It is important to know what we can do and what we cannot do.
